spec/system/casa_admins/edit_spec.rb

```diff
@@ -9,11 +9,13 @@
     it "can successfully edit user email and display name" do
       expected_email = "root@casa.com"
       expected_display_name = "Root Admin"
+      expected_phone_number = "+14163218092"
```

Comment:
This looks too much like a real phone number, please replace it with an obviously fake phone number

Reply:
fixed